docker/develop/Dockerfile

```diff
@@ -66,6 +66,9 @@ ENV CIRCLE_PROJECT_NAME "all-of-us"
 ENV CIRCLE_WORKING_DIRECTORY "${CURATION_HOME}/project/curation"
 ENV CIRCLE_ARTIFACTS "/tmp/circleci-artifacts"
 
+# Enables parallelism in Circle CI but can be 0 if running locally
+ENV CIRCLE_NODE_INDEX "${CIRCLE_NODE_INDEX:-0}"
```

Comment:
Setting this envvar in here isn't functionally useful as it's also set in `define_common_envvars`.  We could also use that to our advantage, as described below.